modules/server/src/main/scala/seqexec/server/gmos/GmosNorthControllerEpics.scala

```diff
@@ -97,6 +97,7 @@ object GmosNorthEncoders extends GmosControllerEpics.Encoders[NorthTypes] {
       case Disperser.B600_G5307  => "B600+_G5307"
       case Disperser.R600_G5304  => "R600+_G5304"
       case Disperser.R400_G5305  => "R400+_G5305"
+      case Disperser.R400_G5310  => "R400-_G5310"
```

Comment:
Unlike the other dispersers, why is it a `-` in this case?

Reply:
That comes from the REL task.

Reply:
Got this from science:
`The value for the new R400 corresponding to your example is R400+_G5310.`